pkg/kubelet/cm/dra/manager_test.go

```diff
@@ -117,7 +117,7 @@ func setupFakeDRADriverGRPCServer(shouldTimeout bool) (fakeDRAServerInfo, error)
 		driverName: driverName,
 	}
 	if shouldTimeout {
-		timeout := plugin.PluginClientTimeout + 10*time.Millisecond
+		timeout := plugin.PluginClientTimeout + 30*time.Millisecond
```

Comment:
WDYT @bart0sh @charles-chenzz ? 30ms seems a bit low for the CI (if it was me I'd go for time.Second)
```suggestion
		timeout := plugin.PluginClientTimeout + 100*time.Millisecond
```

Reply:
I'm ok with it.